samples/ServiceBusSedingToQueue/Program.cs

```diff
@@ -52,17 +52,8 @@ private static async Task MainAsync()
 
             queueClient.RegisteredPlugins.Add(new LogginPlugin(configurations =>
             {
-                configurations.LogginType = LogginType.StorageTable;
-                configurations.SendInBackground = true;
-                configurations.ConfigureStorageAccount(information =>
-                    {
-                        information.ConnectionString = "Insert Storage Connection string";
-                    });
-                configurations.Log = message =>
-                {
-                    // you can handle manually the message here
-                    var m1 = message;
-                };
+                configurations.LogginType = LogginType.Custom;
+                configurations.CustomLogginService = new CustomLogginService();
```

Comment:
This is the challenge with the API I was describing previously.
If I register a custom logging service, why does API force me to provide a redundant piece of information such as `LogginType.Custom`. It should be derived from the fact that I'm registering a custom implementation. Hence the use of an extension method.

Reply:
Yes I agree with that, I think the extension method will help but just changing to avoid the user to configure to explicitly set `Custom` and infer that if CustomLogginService  is not null should enough for now, do't  you think ?

Reply:
It's your call at the end of the day. I'm just describing how I'd feel if would have to provide the same information twice w/o a good reason 🙂

Reply:
yeah and it's very much appreciated ! 💯 🥇

Reply:
Take a look when you have a change of course, finally custom implementation is inferred and an extension method can easily facilitate the implementation for those (And of course getting the implementation)  👍